### ISSUE TITLE
[BUG] My Node Failed to Stop Gracefully.

### ISSUE THREAD

**Issue by jmartinez-43** [quantstamp/qsp-protocol-node#482]
When I tried to stop my node (v2.0.2-alpha2), it "Failed to stop a66042e5bbc0 gracefully."

I attached the logs below: 

[qsp-protocol.log](https://github.com/quantstamp/qsp-protocol-node/files/3249464/qsp-protocol.log)

**Comment by martinetlee** [comment]
Hey @jmartinez-43 , did the following message also appear after "Failed to stop" message and was it able to stop if you apply the `-f` flag?

"Please try running the script again with -f to forcefully stop the container."

As I remember, this is because the Docker wasn't able to stop (waiting for some execution to finish) and is basically benign. I'll check with the team about the actual cause.

**Comment by martinetlee** [comment]
I've consulted with the team, basically what happens here is that the node is waiting for the docker image to stop. Sometimes there are some issues in the docker level, which would take a long time to stop, and after certain time the stop request would expire and you would see the message "Failed to stop gracefully".

However, it is perfectly fine to "Force stop" the node using the `-f` flag as specified by the following message. The only possible thing that could happen here is that if there is a scan assigned to your node, and you've forcefully stopped the node without restarting immediately, the scan request will be expired. As a node-operator, expiring on a scan request would not get slashed. 

In short, as far as we know, it is safe to do the force stop on the node.

**Comment by jmartinez-43** [comment]
Got it
